Add Product Design

teoga opened the pull request:
<!-- Congrats on creating an Awesome list! 🎉 -->

<!-- Please fill in the below placeholders -->

https://github.com/teoga/awesome-product-design

A collection of bookmarks, resources, articles for product designers.

As a product designer you go through many steps such as research, ux, ui, interactions, visuals, ux writing and so on.
This list is intended to have links and articles to guide designers on every step of the way to create their products.

**>>> I've reviewed #1536 and #1538**
 
### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).** Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration.
- I have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title in the format `Add Name of List` (Example: `Add Swift`), not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with the below requirements.


## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- It's the result of hard work and the best I could possibly produce.
- Non-generated Markdown file in a GitHub repo.
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- The image should be high-DPI. Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
		- Can be placed centered if the list has a centered graphics header.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- If you use a license badge, it should be SVG, not PNG.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.

Go to the top and read it again.

sindresorhus commented:
```
❯ awesome-lint https://github.com/teoga/awesome-product-design
✖ Linting

  README.md:79:3
  ✖    1:27  Invalid badge source                      remark-lint:awesome/badge
  ✖   79:3   Link to https://betterdesign.io/ is dead  remark-lint:no-dead-urls
  ✖  179:3   Link to https://handbrake.fr/ is dead     remark-lint:no-dead-urls

  3 errors
```

https://github.com/sindresorhus/awesome-lint

sindresorhus commented on readme.md:
```diff
@@ -606,6 +606,7 @@
 - [Food](https://github.com/jzarca01/awesome-food#readme) - Food-related projects on GitHub.
 - [Mental Health](https://github.com/dreamingechoes/awesome-mental-health#readme) - Mental health awareness and self-care in the software industry.
 - [Bitcoin Payment Processors](https://github.com/alexk111/awesome-bitcoin-payment-processors#readme) - Start accepting Bitcoin.
+- [Product Design](https://github.com/teoga/awesome-product-design) — Links and articles to guide digital product designers.
```
> The entry in the Awesome list should:
> * Include a short description about the project/theme of the list. **It should not describe the list itself.**

teoga replied:
Would something like this work? 
"Helpful resources for digital product designer."

sindresorhus replied:
That still describes the list. It should describe what a digital product design/designer is.

teoga replied:
I just changed it to this
"Research, design, test, learn, iterate."

sindresorhus commented:
> You have to review at least 2 other open pull requests. Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration.

sindresorhus commented:
The header image should not link to your website. It should link to a website that elaborates more about Product Design, like Wikipedia.

sindresorhus commented:
> * **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)

This should explain what product design is and its purpose.

teoga commented:
> https://handbrake.fr

Odd I actually did awesome-lint it 2 minutes ago and handbrake.fr wasn't included. Also I just checked and the link works.
I changed the badge to a new flat-2 and I still get the same lint error 1:27 (not sure exactly why)
Better design is now removed.

sindresorhus commented:
> I changed the badge to a new flat-2 and I still get the same lint error 1:27 (not sure exactly why)

Just ignore it for now. See: https://github.com/sindresorhus/awesome-lint/issues/61

teoga commented:
I also reviewed these 2 #1536 and #1538

sindresorhus commented on readme.md:
```diff
@@ -616,6 +616,7 @@
 - [Amazon Sellers](https://github.com/ScaleLeap/awesome-amazon-seller#readme)
 - [Agriculture](https://github.com/beaorn/awesome-agriculture#readme) - Open source technology for farming and gardening.
 - [Plotters](https://github.com/beardicus/awesome-plotters#readme) - Computer-controlled drawing machines and other visual art robots.
+- [Product Design](https://github.com/teoga/awesome-product-design#readme) — Research, design, test, learn, iterate.
```
This doesn't explain what product design is.

sindresorhus commented:
Tweet: https://twitter.com/awesome__re/status/1126382753035870208
